fcos-graph-builder/src/settings.rs

```diff
@@ -34,10 +34,13 @@ impl ServiceSettings {
     /// Default TCP port for graph-builder main service.
     const DEFAULT_GB_SERVICE_PORT: u16 = 8080;
     /// Default scopes (basearch plus stream) to process.
-    const DEFAULT_SCOPES: [(&'static str, &'static str); 6] = [
+    const DEFAULT_SCOPES: [(&'static str, &'static str); 9] = [
         ("aarch64", "next"),
         ("aarch64", "stable"),
         ("aarch64", "testing"),
+        ("s390x", "next"),
+        ("s390x", "stable"),
```

Comment:
I think this doesn't exist (yet?).
Were you planning to have this PR on hold until we have populated all the stream?

Reply:
correct, stable will get created in two weeks. Does it hurt to have this configured before stable exists? i.e. should we wait and PR that separately in two weeks?

Reply:
Yes, that would be cleaner, otherwise the scraper will be polling-then-erroring for all that time.